README.md

```diff
@@ -103,6 +103,25 @@ Use your favorite method to build it on your system, for example by using
 `aurutils`:
 ```bash
 aur sync kubeone && pacman -S kubeone
+#### Docker
+
+We distribute a docker image that contains KubeOne and Terraform to use for your
```

Comment:
Nit: for now, saying something along `We provide a Dockerfile that you can use to build an image containing KubeOne and Terraform...` could sound better, as we still don't push images.